README.md

```diff
@@ -4,12 +4,14 @@
 
 This cookiecutter provides a template Snakemake profile for
 configuring Snakemake to run on the [SLURM Workload
-Manager](https://slurm.schedmd.com/). The profile defines three
-scripts
+Manager](https://slurm.schedmd.com/). The profile defines the
+following scripts
 
 1. `slurm-submit.py` - submits a jobscript to slurm
 2. `slurm-jobscript.sh` - a template jobscript
 3. `slurm-status.py` - checks the status of jobs in slurm
+4. `slurm-sidecar.py` - run a Snakemake cluster sidecar for caching
```

Comment:
The README should mention somewhere that snakemake v7 is required for sidecar functionality

Reply:
Adding

```
## Dependencies

The Snakemake slurm profile requires Snakemake >=7.0.2 but has no extra dependencies.
```